backend/src/tests/dummyCommunication.test.ts

```diff
@@ -0,0 +1,37 @@
+import supertest from 'supertest';
+import { server } from '../index';
+//import { ApiResponse } from '../types/ApiTypes';
+
+const api = supertest(server);
+
+describe('backend dummy communication, POST /api/textgen', () => {
+    /**
+commented out because it currently fails everytime while the dummy isnt 'connected'
```

Comment:
Don't comment out code, either include it or remove it.

Reply:
removed